.nuget/packages.config

```diff
@@ -21,5 +21,5 @@
   <package id="xunit.runner.console" version="2.2.0" targetFramework="net45" />
   <package id="xunit.runner.msbuild" version="2.2.0" targetFramework="net45" />
   <package id="MicroBuild.Core" version="0.2.0" targetFramework="net45" />
-  <package id="NuGetValidator" version="1.3.7" targetFramework="net461" />
+  <package id="NuGetValidator" version="1.4.0.3" targetFramework="net461" />
```

Comment:
I think he is checking in the package contents just for his testing here, and will remove it eventually